client/lib/wporg/index.js

```diff
@@ -20,18 +20,16 @@ const debug = debugFactory( 'wporg' );
 /**
  * Constants
  */
-let _WPORG_PLUGINS_LIST = 'https://api.wordpress.org/plugins/info/1.1/?action=query_plugins',
-	_DEFAULT_PAGE_SIZE = 24,
-	_DEFAULT_CATEGORY = 'all',
-	_DEFAULT_FIRST_PAGE = 1;
+const _WPORG_PLUGINS_LIST = 'https://api.wordpress.org/plugins/info/1.1/?action=query_plugins';
```

Comment:
I've always wondered why these constants have to be prefixed with an underscore.

Reply:
Just some old convention from Calypso JavaScript medieval times? Underscores removed in 7209b19, together with `options` object mutation.